known-issues.yml

```diff
@@ -0,0 +1,16 @@
+zib-TextResult:
+  zib deviations:
+    DiagnosticReport.code:
+      - cardinality: 1..1
+        reason: The cardinality of both zib and FHIR is 1..1, but because of conceptual cardinality 0..1 is expected in FHIR. Discussion on how to deal with this is ongoing in https://github.com/Nictiz/Nictiz-R4-zib2020/issues/45.
+zib-TextResult-Media:
+  zib deviations:
+    Media.content:
+      - datatype: Attachment
```

Comment:
Shouldn't this be recognized at the profiling guidelines level?

Reply:
Do you mean by adding the Attachment datatype to 'ED'in section 12 Data type mapping? I think so yes!

Reply:
#73

Reply:
Resolved for now/here.